ckan/Dockerfile

```diff
@@ -3,6 +3,7 @@
 FROM debian:jessie
 
 ARG EXTRA_PACKAGES
+ARG PIP_INDEX_URL
```

Comment:
ARG works only for `docker build` phase only. It will not be present in image or container. You still need to declare ENV var for PIP_INDEX_URL to be used in `ckan/post_install_functions.sh` inside the container.

Reply:
I need it only for the build, no?

Reply:
ok, so I assume to propagate in ckan/post_install_functions.sh - got it